far/FarCze.hlf.m4

```diff
@@ -341,6 +341,10 @@ $ #Menu control commands#
  #Alt+Shift+End#
  Align the selected item to the right.
 
+ #Shift+F5#
+ Some menus, e.g., ~all matching entries~@FindAllMenu@ in ~internal editor~@Editor@, have one or more
+fixed columns at the left. The #Shift+F5# key combination toggles visibility of these columns.
```

Comment:
Looks like Shift+F5 toggles it in a more granular way, at least in find all: 0 - 1 - 2 - 0...
If this is intentional and the logic is "each press shows one more column" - maybe describe it here briefly?

Reply:
I thought about it... It's somewhat tedious to describe, while very easy to see and understand. I intentionally did not go into these details. Technically, my wording is correct. It toggles visibility of these columns. Nobody said which columns. 😀

If you think it is worth it, I'll add something. LMK please.

Reply:
Ok, fair enough.